resources/skins/nudus-base/html_style_base.scss

```diff
@@ -211,6 +202,8 @@ pre {
     }
 }
 
+// TODO: Perhaps add kbd
```

Comment:
Sure, why not. :)

Reply:
Yeah, it just so I won't forget.

I'll properly squash commits afterwards, anyhow.